lightly_studio/src/lightly_studio/selection/select_via_db.py

```diff
@@ -67,6 +68,35 @@ def _aggregate_class_distributions(
     return class_distributions
 
 
+def _get_class_balancing_data(
+    strat: AnnotationClassBalancingStrategy,
+    annotations: Sequence[AnnotationBaseTable],
+    input_sample_ids: list[UUID],
+    sample_id_to_annotations: dict[UUID, list[AnnotationBaseTable]],
```

Comment:
Minor: The types could be more general and using a Mapping makes sure that mypy complains about writing to them.

```suggestion
    input_sample_ids: Sequence[UUID],
    sample_id_to_annotations: Mapping[UUID, Sequence[AnnotationBaseTable]],
```